### DISCUSSION TITLE
Add slashes in header menu items links

### DISCUSSION THREAD

**Discussion by epiniguin** [poblabs/weewx-belchertown#804] [Ideas]
Hi!
The header menu items links(Graphs, Records, Reports and About) have no trailing slash.
So it need to add some rules to web-server settings for 301 redirect.
And in this case a browser and a web-server make twice as much work.
In my opinion it would be better to add slashes to these links in the header template.

**Comment by c0by85** [comment]
Hi, you can change by your own. Edit the following lines.

[header.html.tmpl](https://github.com/poblabs/weewx-belchertown/blob/master/skins/Belchertown/header.html.tmpl)

Line 190  
```
<li class="menu-item menu-item-home"><a href="$relative_url/" itemprop="url"><span itemprop="name">$obs.label.nav_home</span></a></li>
```

Line 192 - 196  
```
<li class="menu-item menu-item-1"><a href="$relative_url/graphs/" itemprop="url"><span itemprop="name">$obs.label.nav_graphs</span></a></li>
end if
li class="menu-item menu-item-2"><a href="$relative_url/records/" itemprop="url"><span itemprop="name">$obs.label.nav_records</span></a></li>
li class="menu-item menu-item-3"><a href="$relative_url/reports/" itemprop="url"><span itemprop="name">$obs.label.nav_reports</span></a></li>
li class="menu-item menu-item-4"><a href="$relative_url/about/" itemprop="url"><span itemprop="name">$obs.label.nav_about</span></a></li>
```

**Reply by c0by85** [reply]
Pull request created... https://github.com/poblabs/weewx-belchertown/pull/811
